AppCenterDataStorage/AppCenterDataStorageTests/MSDataStoreTests.m

```diff
@@ -180,4 +257,315 @@ - (void)testPerformCosmosDbAsyncOperationWithHttpClient {
   XCTAssertTrue([expectedURLString isEqualToString:httpClient.sendURL.absoluteString]);
 }
 
+- (void)testCreateWithPartitionGoldenPath {
+
+  // If
+  NSString *expectedDocumentId = @"testDocumentId";
+  NSString *expectedPartition = @"testPartition";
+  id<MSSerializableDocument> mockSerializableDocument = [MSFakeSerializableDocument new];
+  __block BOOL completionHandlerCalled = NO;
+  __block MSDocumentWrapper *actualDocumentWrapper;
+
+  // Mock tokens fetching.
+  MSTokenResult *testToken = [[MSTokenResult alloc] initWithDictionary:[self prepareMutableDictionary]];
+  MSTokensResponse *testTokensResponse = [[MSTokensResponse alloc] initWithTokens:@[ testToken ]];
+  OCMStub([self.tokenExchangeMock performDbTokenAsyncOperationWithHttpClient:OCMOCK_ANY partition:OCMOCK_ANY completionHandler:OCMOCK_ANY])
```

Comment:
DataStore is supposed to call token exchange client with the right partition. Let's make sure of that.
```suggestion
  OCMStub([self.tokenExchangeMock performDbTokenAsyncOperationWithHttpClient:OCMOCK_ANY partition: expectedPartition completionHandler:OCMOCK_ANY])
```

Reply:
Yeah, you're right. Fixed. Thanks.